web/annotation_layer_builder.css

```diff
@@ -140,7 +140,13 @@
    * area, causing horizontal scrolling. We avoid this by extending the width
    * when the element has focus and revert this when it loses focus.
    */
-  width: 115%;
+  /* width: 115%; */
+
+  /* 
+   * Modifying the width because it is making the textAnnotation input box
+   * overlap with other elements  
+   */
```

Comment:
You should never leave commented-out code in patches, and the added comment doesn't really add much value.

Reply:
Thanks for the update.  Yes I have performed all the tests on it, working fine. I'll remove the comment then and the package-lock file.

Reply:
Please note https://github.com/mozilla/pdf.js/wiki/Squashing-Commits

Given that all of this code was written by Tim, I think it makes sense to ask him to review it (time permitting).